New admin method for fetching group offsets for multiple topics (#992)
tulios/kafkajs: docs/Admin.md

## <a name="fetch-offsets"></a> Fetch consumer group offsets

`fetchOffsets` returns the consumer group offset for a list of topics.

```javascript

> await admin.fetchOffsets({ groupId, topics, })

Reviewer: Removing trailing comma and making the input match the example output.

```suggestion
await admin.fetchOffsets({ groupId, topics: ['topic1', 'topic2'] })
```

// [
//   {
//     topic: 'topic1',
//     partitions: [
//       { partition: 0, offset: '31004' },
//       { partition: 1, offset: '54312' },
//       { partition: 2, offset: '32103' },
//       { partition: 3, offset: '28' },
//     ],
//   },
//   {
//     topic: 'topic2',
//     partitions: [
//       { partition: 0, offset: '1234' },
//       { partition: 1, offset: '4567' },
//     ],
//   },
// ]
```

> Omit `topics` altogether if you want to get the consumer group offsets for all of its consumed topics.

Reviewer: ```suggestion
Omit `topics` altogether if you want to get the consumer group offsets for all topics with committed offsets.
```
